DEVELOPING.md

```diff
@@ -24,11 +24,6 @@ TAG=my-tag
 ./scripts/build.sh gcr.io/$PROJECT_ID $TAG
```

Comment:
Mention which permissions must be granted to the `cloudbuild.gserviceaccount.com` service account:
* App Engine Admin
* Logs Viewer
* Container Analysis Admin
* Others??
...And that the "Google Service Management API" needs to be enabled on the project.